### DISCUSSION TITLE
Auto text dropped in version 2.7.8.  Can I get it back?

### DISCUSSION THREAD

**Discussion by jtreagan** [keepassxreboot/keepassxc#11415] [Q&A] [answered]
So, I updated my KeepassXC to version 2.7.8 and the Autotext fill feature is gone.  Can I get it back or should I revert to an older version?

**Comment by droidmonkey** [comment, marked as answer]
You are running under Wayland now and can't use auto-type

**Comment by jtreagan** [comment]
So, how do I get rid of Wayland, whatever it is?

**Reply by droidmonkey** [reply]
https://github.com/keepassxreboot/keepassxc/issues/2281#issuecomment-2275246478

**Comment by jtreagan** [comment]
@droidmonkey I did follow that link and tried to read through that thread, but it got kind of crazy and I didn't find a good solution.  Thanks for trying!  :>)  However, I did discover this:  When first starting Ubuntu 24.04, before entering user/password click the tiny icon that appears in the lower right corner of the lock screen.  That will allow you to choose X11 instead of Wayland and then KeepassXC will work better.  The autotext feature is kind of clumsy in this version, but it works.
